### PR TITLE
Upgraded issue template form

### PR DESCRIPTION
**Pull request by bipinparajuli**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/documentation.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  - type: checkboxes
+    id: no-duplicate-issues
+    attributes:
+      label: "👀 Have you spent some time to check if this issue has been raised before?"
+      description: "Have you Googled for a similar issue or checked our older issues for a similar bug?"
+      options:
+        - label: "I checked and didn't find similar issue"
+          required: true
+  - type: checkboxes
+    id: read-code-of-conduct
```
Comment (lohanidamodar) [inline .github/ISSUE_TEMPLATE/documentation.yaml:27]: Please add this code of conduct option to all the forms. Thank you.

**Review by lohanidamodar — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/documentation.yaml (hunk 1 above)
Comment (bipinparajuli) [reply .github/ISSUE_TEMPLATE/documentation.yaml:27]: can you please recheck it. I can see checkbox for code of conduct on each form

**Comment by lohanidamodar** [comment]
@bipinparajuli  Great work. One more change before we can approve. We have made few changes in the language and also the format of Code of conduct. Please check https://github.com/appwrite/appwrite template and update accordingly.
Thank you.

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    validations:
+      required: true
+    attributes:
+      label: "👎 Actual Behavior"
+      description: "What did actually happen? Add screenshots, if applicable."
+      placeholder: "It actually ..."
+  - type: dropdown
+    id: appwrite-version
+    attributes:
+      label: "🎲 Appwrite version"
```
Comment (lohanidamodar) [inline .github/ISSUE_TEMPLATE/bug.yaml:37]: Please use `Docker Resque Version` and list corresponding version from the releases.

**Review by lohanidamodar — COMMENTED** [review]
@bipinparajuli one small change 👍🏻

**Comment by christyjacob4** [comment]
@bipinparajuli  let us know if you're still working on this PR, else we'll need to close it 👍

**Comment by bipinparajuli** [comment]
> @bipinparajuli let us know if you're still working on this PR, else we'll need to close it +1

@christyjacob4 changes have been completed but 

> @bipinparajuli let us know if you're still working on this PR, else we'll need to close it +1

@christyjacob4 changes have been completed but the build failed

**Comment by bipinparajuli** [comment]
> @bipinparajuli let us know if you're still working on this PR, else we'll need to close it +1

@christyjacob4 changes have been completed but the build failed can you please help me to resolve

**Review by Meldiron — APPROVED** [review]
Looks good to me

**Comment by Meldiron** [comment]
THANK YOU! All changes merged 🥳

Please reach out to `eldadfux` on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+is%3Aissue+label%3Ahacktoberfest+created%3A%3E2021-01-01+is%3Aopen&type=issues) and help make Appwrite better :)

**Pull request merged**